org.eclipse.jdt.core.tests.model/src/org/eclipse/jdt/core/tests/model/JavaSearchBugsTests2.java

```diff
@@ -2894,6 +2894,54 @@ public void testOutOfBoundsIndexExceptionDuringSearchGh825() throws Exception {
 			deleteProject(projectName);
 		}
 	}
+	public void testCallHierarchyAnonymousInnerTypeGh856() throws CoreException, IOException {
+		try {
+			IJavaProject p = createJavaProject("P", new String[] { "src" }, new String[] { "JCL11_LIB" }, "bin", "11");
+			createFolder("/P/src/test");
+			createFile("/P/src/test/X.java", String.join(System.lineSeparator(), new String[] {
+					"package test;\n",
+					"public class X {\n",
+					"    public static interface Action1 {\n",
```

Comment:
what if we have a method or a field definition before the first member type definition ? does this test pass still. I'm working about the usage of `type.getOccurenceCount()` ?